{"id": "5340", "title": "Bug 5340: Cancelling add exception breakpoint has no effect", "description": "", "OB": "When adding an exception breakpoint in the debug perspective, canceling the operation does not have any effect. The progress dialog immediately closes, and the exception list comes up ignoring the cancellation request. ", "EB": "The cancellation request should be honored, and the progress dialog should close without displaying the exception list. Alternatively, the cancel button should be disabled when adding an exception breakpoint. To achieve this, when calling the ProgressMonitorDialog.run method, the value of the 'cancelable' parameter should be set to false.", "SR": "To reproduce the issue: \n1) Go to the breakpoints pane in the debug perspective. \n2) Click the J! button to add an exception. \n3) In the progress dialog, click cancel."}